src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -235,10 +304,10 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
 
     ***************************************************************************/
 
-    private static void notify ( ref UserSpecifiedParams params,
+    private static void notify ( UserSpecifiedParams* params,
```

Comment:
Just for convenience. `user_params` returns a pointer to a struct now, so usage is simpler if `notify` accepts the same type.